docs/bank-accounts/Basics.md

```diff
@@ -0,0 +1,53 @@
+
+Below you can see a state diagram with possible state transitions, followed by a table to explain what is meant by each state.
+
+```mermaid
+stateDiagram-v2
+    [*] --> Pending
+    Pending --> Active
+    Pending --> Suspended
+    Pending --> Inactive
+    Active --> Suspended
+    Active --> Inactive
+    Suspended --> Active
+    Suspended --> Inactive
+    Inactive --> [*]
+```
+
+| Status | Explanation |
+|---|---|
+| Pending | A short-lived state where the bank account can't yet be used for funding, as checks are being performed on the bank account. |
+| Active | The bank account is ready to be used for funding. |
+| Suspended | One or more checks on the bank account have failed and the bank account is suspended. Once checks have been fixed the account will move back to `Active`. |
+| Inactive | The bank account is no longer active and cannot be used for funding. To use the bank account again, it will need to be added as a new bank account. |
+
+# Bank account checks
+
+WealthKernel perform checks on bank accounts to makes sure that the bank account details provided match the party name. It is important that the correct party name and bank details are provided, otherwise these checks are likely to fail and require manual intervention.
+
+In the rare cases where we cannot automatically verify a bank account belongs to the party, we may need additional information to prove that the bank account belongs to them. The bank account status will remain in a `Pending` state until the additional information has been provided and verification has been successful. To receive the most up-to-date information about a bank account's status, we recommend using the bank account webhooks which are detailed below.
+
+# Bank account webhooks
+
+You can opt in to receive bank accounts webhooks which will give you updates about bank accounts' current status. You can find out more about using webhooks on our [webhooks guide](../webhooks/Getting-Started.md), or look at our main API documentation to see the structure of each of the bank accounts webhooks.
+
+Currently, the following webhooks are available:
+
+| Event Type | Description |
+|------------|------------:|
+| `bank_accounts.bank_account_activated` | The bank account has passed checks and can be used for funding. The bank account will be in the `Active` state. |
+| `bank_accounts.bank_account_suspended` | One or more checks on the bank account have failed and the bank account is suspended. We will not accept funding, and the bank account will be in the `Suspended` status. |
+| `bank_accounts.bank_account_deactivated` | The bank account has been deactivated and can no longer be used for funding. The bank account will be in the `Inactive` state. |
```

Comment:
Is this duplication of what's on the webhooks page? If so, I worry it might go out of date if we change anything in the webhooks page

Reply:
Webhooks page? We have these listed on our main API reference under the [schemas](https://docs.wealthkernel.com/docs/api/1f9ca0dc0c07f-bank-account-activated-v1-webhook) but they aren't listed under any of our other guides. 
This page mimics the one we have for accounts, which [also lists webhooks](https://docs.wealthkernel.com/docs/guides/b3b8b172d6677-account-basics#accounts-webhooks). I don't think there's anything for it to go out of sync with.